lib/vf_actions/src/builtins.rs

```diff
@@ -7,52 +7,52 @@
  *
  * @see https://github.com/valueflows/valueflows/issues/487#issuecomment-482161938
  */
-use super::{
-    Action,
-    ActionEffect,
-    ProcessType,
-};
+use super::{Action, ActionEffect, ProcessType};
 
 // setup for core actions as in-memory statics
 
 macro_rules! generate_builtin_actions {
-    ($key: expr; $( $a:ident => $e:expr, $f:expr, $g:expr );*) => {
+    ($key: expr; $( $a:ident => $e:expr, $f:expr, $g:expr, $h:expr );*) => {
         match &str::replace($key, "-", "_")[..] {
             $(
                 stringify!($a) => Some(Action {
                     id: str::replace(stringify!($a), "_", "-"),
                     label: str::replace(stringify!($a), "_", "-"),
                     resource_effect: $e,
```

Comment:
@fosterlynn this is where it's called "resource_effect"

Reply:
Not sure what that hunk of code actually does. :sweat_smile: .  Here is what I'm in process of putting in to the vf doc that you care about:

The economic resource's accounting quantity and onhand quantity are both affected (decremented or incremented or no effect) by the resource quantity of the economic event.

Hope that helps you sort it out!